q2_diversity_lib/tests/test_alpha.py

```diff
@@ -32,56 +30,89 @@ def setUp(self):
 
     def test_non_phylogenetic_passed_empty_table(self):
         for measure in nonphylogenetic_measures:
-            with self.assertRaisesRegex(ValueError, "empty"):
+            with self.assertRaisesRegex(ValueError, 'empty'):
                 measure(table=self.empty_table)
 
 
 class FaithPDTests(TestPluginBase):
-
     package = 'q2_diversity_lib.tests'
 
     def setUp(self):
         super().setUp()
-        self.input_table = biom.Table(np.array([[1, 0, .5, 999, 1],
-                                                [0, 1, 2, 0, 1],
-                                                [0, 0, 0, 1, 1]]),
-                                      ['A', 'B', 'C'],
-                                      ['S1', 'S2', 'S3', 'S4', 'S5'])
-        self.input_tree = skbio.TreeNode.read(io.StringIO(
-                '((A:0.3, B:0.50):0.2, C:100)root;'))
-        self.faith_pd_expected = pd.Series({'S1': 0.5, 'S2': 0.7, 'S3': 1.0,
-                                            'S4': 100.5, 'S5': 101},
-                                           name='faith_pd')
+        self.empty_table_fp = self.get_data_path('empty_table.biom')
+        self.empty_table_as_BIOMV210Format = \
+            BIOMV210Format(self.empty_table_fp, mode='r')
```

Comment:
Assigning this filepath as an instance property implies that it is used (or intended for use) in other methods (read: tests). It will be more clear to future readers if you are clear about the scope of this variable

```suggestion
        empty_table_fp = self.get_data_path('empty_table.biom')
        self.empty_table_as_BIOMV210Format = \
            BIOMV210Format(empty_table_fp, mode='r')
```

Please propagate these changes to your other test suites, too.